cpp/src/io/parquet/reader_impl_helpers.cpp

```diff
@@ -537,6 +558,214 @@ aggregate_reader_metadata::aggregate_reader_metadata(
       CUDF_EXPECTS(schema == pfm.schema, "All sources must have the same schema");
     }
   }
+
+  if (arrow_schema.has_value()) {
+    // erase "ARROW:schema" from the output pfm.
```

Comment:
Why remove the arrow schema? Won't this cause trouble downstream when we write the duration back out as millisconds?

Reply:
We need to remove this as this has no further role beyond this point and shouldn't be used. When writing back, we should construct a fresh schema structure (as the columns or data might have changed) and encode and write that in the output parquet file instead. Also, arrow removes this as well.

Reply:
@vuule for vis